tests/gpflow/test_functions.py

```diff
@@ -234,8 +235,8 @@ def test_polynomial__1d() -> None:
     Y = p(X)
     assert_allclose(
         [
-            [1.0 + 2.0 * 1.0 + 3.0 * (1.0 ** 2)],
-            [1.0 + 2.0 * 2.0 + 3.0 * (2.0 ** 2)],
+            [1.0 + 2.0 * 1.0 + 3.0 * (1.0**2)],
```

Comment:
This change suggests that you have the wrong version of black installed! To ensure `make format` works you should first run `make dev-install` which should install it. (I'm not sure it actually says this anywhere in CONTRIBUTING.md, sorry! I'll make sure to update the instructions.)

To check which version of black you have, run `pip show black`. The current version used by the tests is 20.8b1.